CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # CHANGELOG
 
+## NEXT RELEASE
```

Comment:
In the future, we can add these separate when we go to release since it's not related to the title of this PR.

Reply:
sure